data-transfer-object-enum-impl/src/main/java/com/iluwatar/datatransferenum/Product.java

```diff
@@ -0,0 +1,91 @@
+package com.iluwatar.datatransferenum;
+
+/**
+ * {@link Product} is a entity class for product entity. This class act as entity in the demo.
+ */
+public final class Product {
+  private Long id;
+  private String name;
+  private Double price;
+  private Double cost;
+  private String supplier;
+
+  /**
+   * Constructor.
+   *
+   * @param id       product id
+   * @param name     product name
+   * @param price    product price
+   * @param cost     product cost
+   * @param supplier product supplier
+   */
+  public Product(Long id, String name, Double price, Double cost, String supplier) {
+    this.id = id;
+    this.name = name;
+    this.price = price;
+    this.cost = cost;
+    this.supplier = supplier;
+  }
+
+  /**
+   * Constructor.
+   */
+  public Product() {
+  }
+
+  public Long getId() {
```

Comment:
A suggestion. You can use Lombok packages from this repo which avoids all the boilerplate code

Reply:
agree with you @ravening